### ISSUE TITLE
Table resize plugin doesn't work due to being blocked by a focus trap.

### ISSUE THREAD

**Issue by rickygao** [microsoft/roosterjs#1804]
**Describe the bug**

Table resize plugin doesn't work due to being blocked by a focus trap.

**To Reproduce**

1. Use an editor with `TableResize` plugin in a `FocusTrap` (fluentui v8)
2. Click on any row/column inserters.
3. Observe the issue.

**Expected behavior**

Should work.

**Screenshots**

**Device Information**
 - OS: Windows 11 Enterprise 22H2
 - Browser: Microsoft Edge for Business
 - Version:113.0.1774.42 (Official build) (64-bit)

**Additional context**

This could be due to the return value of `createElement` is append to `document.body`, when trying to insert inserters, regardless the context.

**Comment by JiuqingSong** [comment]
Hi, thanks for reporting this issue. If you have a fix, would you like to submit your fix to the repo?

Thanks.

**Comment by JiuqingSong** [comment]
Hi @rickygao , do you want to submit a fix? Or I'll close this issue for now since we don't plan to fix it from our side. Thanks

**Comment by rickygao** [comment]
> Hi @rickygao , do you want to submit a fix? Or I'll close this issue for now since we don't plan to fix it from our side. Thanks

Please feel free to close this issue, thanks! This is not a priority for us and may not be fixed in the near future.
